packages/connect-examples/expo-playground/app/components/sidebar.tsx

```
@@ -128,6 +130,17 @@ export function AppSidebar() {
                           {getStatusText()}
                         </Badge>
                       </div>
+                      {/* 设备详情按钮 - 优化设计 */}
+                      <button
+                        onClick={() => navigate('/device-info')}
+                        className="group relative flex items-center gap-2 px-3 py-2 rounded-lg bg-background hover:bg-accent text-foreground hover:text-accent-foreground border border-border hover:border-border transition-all duration-200 hover:shadow-sm"
+                        title="View Device Details"
+                      >
+                        <div className="flex items-center gap-2">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)